### PR TITLE
New version: Kakao.KakaoTalk version 3.6.5.3663

### PR DESCRIPTION
**Pull request by MokaSoda**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124959)

### CODE REVIEW

**Comment by MokaSoda** [comment]
@microsoft-github-policy-service agree

**Comment by MokaSoda** [comment]
@microsoft-github-policy-service rerun

**Comment by MokaSoda** [comment]
@microsoft-github-policy-service agree

**Comment by BrandonWanHuanSheng** [comment]
@MokaSoda There is something wrong with the @microsoft-github-policy-service. Because @microsoft-github-policy-service did not add or remove the labels on some of the PR. Please do not close this pull request. There might need some fix to applied https://github.com/microsoft/winget-pkgs/tree/master/.github/workflow. It seem like there is some labels need to re-fix it. The team is already in aware.

**Comment by MokaSoda** [comment]
@microsoft-github-policy-service agree

**Comment by MokaSoda** [comment]
@microsoft-github-policy-service rerun

**Comment by BrandonWanHuanSheng** [comment]
@mdanish-kh. What is happening here?

**Comment by mdanish-kh** [comment]
The bot appears to be down today. This will need to be looked at by the maintainers.

**Comment by denelon** [comment]
It looks like the bot approved merging 10 hours ago. I'm going to trigger another run to see if it's resolved.

File: manifests/k/Kakao/KakaoTalk/3.6.5.3663/Kakao.KakaoTalk.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Kakao.KakaoTalk
+PackageVersion: 3.6.5.3663
```
Comment (mdanish-kh) [inline manifests/k/Kakao/KakaoTalk/3.6.5.3663/Kakao.KakaoTalk.installer.yaml:5]:
```suggestion
PackageVersion: 3.6.6.3669
```

File: manifests/k/Kakao/KakaoTalk/3.6.5.3663/Kakao.KakaoTalk.locale.en-US.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Automatically updated by the winget bot at 2023/May/18
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
 
 PackageIdentifier: Kakao.KakaoTalk
-PackageVersion: 3.5.1.3469
+PackageVersion: 3.6.5.3663
```
Comment (mdanish-kh) [inline manifests/k/Kakao/KakaoTalk/3.6.5.3663/Kakao.KakaoTalk.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.6.6.3669
```

File: manifests/k/Kakao/KakaoTalk/3.6.5.3663/Kakao.KakaoTalk.locale.ko-KR.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Automatically updated by the winget bot at 2023/May/18
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.2.0.schema.json
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.5.0.schema.json
 
 PackageIdentifier: Kakao.KakaoTalk
-PackageVersion: 3.5.1.3469
+PackageVersion: 3.6.5.3663
```
Comment (mdanish-kh) [inline manifests/k/Kakao/KakaoTalk/3.6.5.3663/Kakao.KakaoTalk.locale.ko-KR.yaml:5]:
```suggestion
PackageVersion: 3.6.6.3669
```

File: manifests/k/Kakao/KakaoTalk/3.6.5.3663/Kakao.KakaoTalk.yaml [hunk 4]
```diff
@@ -1,8 +1,8 @@
-# Automatically updated by the winget bot at 2023/May/18
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
 
 PackageIdentifier: Kakao.KakaoTalk
-PackageVersion: 3.5.1.3469
+PackageVersion: 3.6.5.3663
```
Comment (mdanish-kh) [inline manifests/k/Kakao/KakaoTalk/3.6.5.3663/Kakao.KakaoTalk.yaml:5]:
```suggestion
PackageVersion: 3.6.6.3669
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
PackageVersion must match version displayed in Apps&Features / Control Panel

After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/k/Kakao/KakaoTalk` should be renamed from `3.6.5.3663` -> `3.6.6.3669`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by BrandonWanHuanSheng** [comment]
@MokaSoda There is some changes requested to your bot.

**Comment by MokaSoda** [comment]
Okay, I'll fix this by running wingetcreate again. Thanks for solving CLA and mentioning version problem.

**Pull request closed without merging**